napari/_qt/menus/plugins_menu.py

```diff
@@ -59,7 +59,15 @@ def _build(self, event=None):
                     full_name = menu_item_template.format(*key)
                     action = QAction(full_name, parent=self)
 
-                def _add_widget(*args, key=key, hook_type=hook_type):
+                def _add_toggle_widget(*args, key=key, hook_type=hook_type):
```

Comment:
ALso at the same time, maybe be more explicit if you know what is in `*args`. If nothing is passed, maybe just `*`, to make it clear no pos args ar passed.